Linker error with HeatinSystem model

Today I tried to run `Modelica.Fluid.Examples.HeatingSystem` in OMEdit  with a recent nightly build. I got the following linker error
```
C:\PROGRA~1\OPENMO~1.0-D\tools\msys\mingw64\bin\ld: HeatingSystem_functions.o:HeatingSystem_func:(.text+0xab72): undefined reference to `dgesv_'
clang: error: linker command failed with exit code 1 (use -v to see invocation)
mingw32-make: *** [HeatingSystem.makefile:37: omc_main_target] Error 1
mingw32-make: Leaving directory 'C:/Users/FRANCE~1/AppData/Local/Temp/OPENMO~1/OMEdit/MODELI~1.HEA'
Compilation process failed. Exited with code 0x00000002.
```
I'm not sure exactly what the problem is, but it looks quite bad. Not sure when this started, either. I can do some bisection on downloaded installation packages. I suspect this is a windows-specific issue, possibly related to short DOS filenames?

### Version and OS

 - OpenModelica Version:  (v1.20.0-dev-326-gf2a0f69834-64bit on Windows
 - OS: Windows 10, 64 bit

--- a/OMCompiler/Compiler/Util/Autoconf.mo.omdev.mingw
+++ b/OMCompiler/Compiler/Util/Autoconf.mo.omdev.mingw
@@ -22,7 +22,7 @@ encapsulated package Autoconf
                                   "-luser32 -lkernel32 -ladvapi32 -lshell32 -lopenblas -Wl,-Bdynamic";
 
   constant String ldflags_runtime = " -lOpenModelicaRuntimeC" + ldflags_basic;
-  constant String ldflags_runtime_sim = "-lSimulationRuntimeC -Wl,-Bdynamic -lomcgc " + linkType + " -lstdc++ -Wl,-Bdynamic ";
+  constant String ldflags_runtime_sim = "-lSimulationRuntimeC -Wl,-Bdynamic -lomcgc -lopenblas" + linkType + " -lstdc++ -Wl,-Bdynamic ";
   constant String ldflags_runtime_fmu = linkType + "-lregex -ltre -lintl -liconv -static-libgcc -lpthread -lm " +
                                         libFortran +
                                         "-lgfortran -lquadmath -lmingw32 -lgcc_eh -lmoldname -lmingwex " +

